server/src/server.ts

```diff
@@ -6527,553 +6359,130 @@ Email verified! You can close this tab or hit the back button.
   }
 
   const getUser = User.getUser;
+  const getComments = Comment.getComments;
+  const _getCommentsForModerationList = Comment._getCommentsForModerationList;
+  const _getCommentsList = Comment._getCommentsList;
+  const getNumberOfCommentsRemaining = Comment.getNumberOfCommentsRemaining;
 
-  // These map from non-ui string codes to number codes used in the DB
-  var planCodes = {
-    mike: 9999,
-    trial: 0,
-    free: 0,
-    individuals: 1,
-    students: 2,
-    pp: 3,
-    sites: 100,
-    org99: 99,
-    pro: 300,
-    organizations: 1000,
-  };
+  /*
+   Rename column 'zid' to 'conversation_id', add a new column called 'zid' and have that be a VARCHAR of limited length.
+   Use conversation_id internally, refactor math poller to use conversation_id
+   continue to use zid externally, but it will be a string of limited length
+   Don't expose the conversation_id to the client.
```

Comment:
Uhh... what is this comment referring to here? The "string" part of the url is `zinvite` and is in a separate table. As much as we might quibble about specific choice of `zid`, there is actually a pretty consistent primary key naming pattern throughout the app so I'd like to keep that. I _do_ see that there's code in here that uses `conversation_id`, which is fine, but I think the tables should probably stay as-is.

Reply:
Not sure what happened to make this diff look like this, but this seems to already be in `edge`, so must just be a reorg issue.

Reply:
Still this comment is way out of date, and the thing that is suggested would happened here never happened, so going to remove.